### PR TITLE
Enriched the model with some more constraints

### PR DESCRIPTION
**Pull request by rjgorthi**
Enhanced the model to depict a better use of if/else constructs.

### CODE REVIEW

**Comment by vivshankar** [comment]
@rjgorthi please clarify what has changed. It's hard to parse the differences in the XML

**Comment by rjgorthi** [comment]
@vivshankar - Enhanced the if-else constructs to fork out to different IDPs. Earlier, the model was CD or invalid! Now including more options, just to make the model richer! Also gives a clear understanding how to do nested if-else with the rules! Have also updated the corresponding blog!

**Comment by rjgorthi** [comment]
@vivshankar - Just to add one more point. The core model is not changing. It's a change only in the "Compute Login URL" Custom Task. Just expanded onto that rule a bit, to do some more if-else nesting, so that it becomes more consumable.

**Review by vivshankar — APPROVED** [review]
